exercise-1.js

```diff
@@ -385,7 +537,8 @@
         // Find all devs older than 24
 
         //your code here...
-
+        let greaterThan24 = devs.filter(({age}) => age >= 24);
```

Comment:
Good to see an Array method was used here -- I was hoping you hadn't completely missed those.